presto-tests/src/test/java/io/prestosql/execution/TestEventListener.java

```diff
@@ -262,12 +263,16 @@ private void assertFailedQuery(Session session, @Language("SQL") String sql, boo
 
     private EventFilters buildEventFilters(boolean ignoreSplits)
     {
+        Instant now = Instant.now();
         EventFilters.Builder builder = EventFilters.builder()
-                .setQueryCreatedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER))
-                .setQueryCompletedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER));
+                .setQueryCreatedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER) && !event.getCreateTime().isBefore(now))
```

Comment:
GHA clock is not reliable. https://github.com/prestosql/presto/issues/5608

can we have marker per test instead?

Reply:
Good idea. I like it much more that the clock.

Reply:
Did not get that pretty :/